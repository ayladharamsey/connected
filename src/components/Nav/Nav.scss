nav {
    background-color: #6B7A8F;
    height: 100vh;
}

h3 {
    padding-left: 20px;
}

.links {
    margin-top: 70px;
}

a {
    text-decoration: none;
}

.links {
        h4 {
            color: #FFFFFF;
            font-family: 'Playfair Display';
            font-size: 1.2em;
            margin-left: 20px;
        }

        .hover {
            padding: 10px;
            display: flex;
            margin-top: 30px;
            &:active, &:hover{
                padding: 10px;
                background-color: #C17538;
            }
        }

        img {
            height: 24px;
            margin-left: 20px;
        }
}

