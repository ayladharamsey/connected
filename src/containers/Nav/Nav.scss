nav {
    background-color: #6B7A8F;
    height: 100vh;
}

h3 {
    padding-left: 20px;
}

.links {
    margin-top: 70px;
}

a {
    text-decoration: none;
}

h4 {
    color: #FFFFFF;
    padding: 20px;
    font-family: 'Playfair Display';
    font-size: 1.3em;
    &:active, &:hover{
        background-color: #C17538;
    }
}