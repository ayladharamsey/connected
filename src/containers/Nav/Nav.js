import React from 'react';
import { Link } from 'react-router-dom';
import './Nav.scss'; 
import feed from '../../images/edited/newsfeed-white.png';
import clock from '../../images/edited/clock-white.png';
import check from '../../images/edited/check-white.png';

export const Nav = ({ filterByNav }) => {
    
    return (
        <nav>
            <h1>connected</h1>
            <h3>your personalized news feed</h3>
            <div className="links" >
                <Link to='/' onClick ={filterByNav}>
                    <div className="hover">
                        <img src={feed}/>
                        <h4 className="live" >Live Feed</h4>
                    </div>
                </Link>
                <Link to='/saved' onClick ={filterByNav}>
                    <div className="hover">
                        <img src={clock}/>
                        <h4 className="saved">Saved For Later</h4>
                    </div>
                </Link>
                <Link to='/read' onClick ={filterByNav}>
                    <div className="hover">
                        <img src={check}/>
                        <h4 className="completed">Completed Articles</h4>
                    </div>
                </Link>
            </div>
        </nav>
    )
}

