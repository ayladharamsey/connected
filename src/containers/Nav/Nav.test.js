import React from 'react';
import { shallow } from 'enzyme';
import { Nav } from './Nav';


describe( 'Nav', () => {
    const filterByNav = jest.fn();
    let wrapper;
    
    beforeEach(() => {
        wrapper = shallow(<Nav filterByNav={filterByNav}/>)
    })
    
    it('should match the snapshots', () => {
        expect(wrapper).toMatchSnapshot();
    })

})