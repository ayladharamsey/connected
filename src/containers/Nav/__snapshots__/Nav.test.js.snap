// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshots 1`] = `ShallowWrapper {}`;
